concourse/scripts/pipelines-cloudfoundry.sh

```diff
@@ -93,6 +93,7 @@ pipeline_name: ${pipeline_name}
 enable_destroy: ${ENABLE_DESTROY:-}
 self_update_pipeline: ${SELF_UPDATE_PIPELINE:-true}
 aws_account: ${AWS_ACCOUNT:-dev}
+makefile_env_target: ${MAKEFILE_ENV_TARGET}
```

Comment:
On line 76 and 82 we should replace AWS_ACCOUNT with MAKEFILE_ENV_TARGET

Reply:
Fixed.